pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/imports/UnnecessaryFullyQualifiedNameRule.java

```diff
@@ -32,11 +33,6 @@ public UnnecessaryFullyQualifiedNameRule() {
     @Override
     public Object visit(ASTCompilationUnit node, Object data) {
         imports.clear();
-        violations.clear();
-
-        super.visit(node, data);
```

Comment:
I took the chance to kill this. Calling `super.visit` in a `RuleChainVisitor` ends up with the other visit methods being called twice, and errors being duplicated.